packages/amplify_auth_cognito/example/tool/provision_integration_test_resources.sh

```diff
@@ -17,9 +17,65 @@ FRONTEND="{\
 \"config\":$FLUTTERCONFIG\
 }"
 
+# read the request template and the schema
```

Comment:
I was noticing that this script works well when I run on normal aws account w default profile, but it does not work well with an isengard account or other profile. To account for that, I have a request: make this script accept an env variable AWS_PROFILE, then have the Amplify headless and AWS CLI refer to that variable (default to "default").

At the top, will look like 
```
# In development, AWS_PROFILE should be set. In CI, it's not.
[ "$AWS_PROFILE" ] && profileName="$AWS_PROFILE" || profileName="default";

FLUTTERCONFIG="{\
\"ResDir\":\"./lib/\",\
}"

AMPLIFY="{\
\"projectName\":\"amplifyauthinteg\",\
\"envName\":\"test\",\
\"defaultEditor\":\"code\"\
}"

FRONTEND="{\
\"frontend\":\"flutter\",\
\"config\":$FLUTTERCONFIG\
}"

AWSCLOUDFORMATIONCONFIG="{\
\"configLevel\":\"project\",\
\"useProfile\":"true",\
\"profileName\":\"$profileName\",\
\"region\":\"us-west-2\"\
}"
PROVIDERS="{\
\"awscloudformation\":$AWSCLOUDFORMATIONCONFIG\
}"
```

Then should use that profile as a `--profile` argument to all the AWS CLI commands to ensure everything goes through same profile.